README.md

```diff
@@ -0,0 +1,16 @@
+# parity-instantseal
+Builds instantseal Docker container, normally available at keydonix/parity-instantseal.
+
+Comes funded with these accounts:
+- 0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb (private key: 0xfae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5an), loaded with 10^20 ETH
```

Comment:
Do you mean 2^128 ETH (which is about 10^38.5)?

Reply:
yea, figure it was more useful for README to report it in ETH terms

Reply:
Ah, misread as attoeth.  👍